src/engine/strat_engine/filesystem.rs

```diff
@@ -145,24 +145,40 @@ impl StratFilesystem {
     /// Return an extend size for the thindev under the filesystem
     /// TODO: returning the current size will double the space provisoned to
     /// the thin device.  We should determine if this is a reasonable value.
+    ///
```

Comment:
Accidental extra line introduced.

Reply:
@mulkieran  Ugh, not sure how that got in there, thanks!  PR spun again.